src/main/java/net/dv8tion/jda/internal/audio/AudioConnection.java

```diff
@@ -468,7 +468,7 @@ else if (couldReceive)
             receiveThread.start();
         }
 
-        if (receiveHandler.canReceiveCombined())
+        if (receiveHandler.setCombinedExecutor())
```

Comment:
Is there a reason for why this method follows a setter like naming scheme while not actually being a setter?

Reply:
fixed